### PR TITLE
So/leo adef

### LINKED ISSUE #10: Project 8: Leo Adef (photography articles)
image

### LINKED ISSUE #363: Add February image to Leo Adef calendar

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?
- all images have now been added to the calendar - Jan to December
- I've added scrollback functionality to the calendar. Since we're using the same scrollback as on the Projects page, I've created a component named `Scrollback`
- Scrollback styled-components have been removed and replaced with the imported React F.C `Scrollback` 
- There's a slight variation on where the button should be placed on the Projects and the Leo Adef page so I've passed the component some props called `bottom` and `left` 

### Which issue(s) does this PR fix?

Fixes #10 
Fixes #363 

### Screenshots (if there are design changes)
Screenshot 2021-01-25 at 17 22 59

### How to test
- Set the date back to a date in the past and the content should appear
- Check that there are 12 images, one for each month 
- Check that the scrollback works on mobile (nb display: none on larger screen sizes)

### CODE REVIEW

File: src/components/Scrollback.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+const Scrollback: React.FC<ButtonProps> = props => {
+  const { bottom, left } = props;
+  const scrollToTop = useCallback(() => {
+    if (window) {
+      window.scrollTo({ top: 0, behavior: "smooth" });
+    }
+  }, []);
+  return (
+    <Fragment>
```
Comment (Jbarget) [inline src/components/Scrollback.tsx:39]: i dont think you need the `Fragment`, you only really need it when there are 2 jsx elements side by side

eg. if Scrollback was:

```
return (
  <Fragment>
    <div></div>
    <Button />
  </Fragment>
)
```

File: src/components/Projects.tsx [hunk 2]
```diff
@@ -54,18 +36,7 @@ const Projects = () => {
...
-        bottom={6}
-        left={6}
-        fontSize={[3, 4]}
-        background="transparent"
-        onClick={scrollToTop}
-        zIndex={theme.zIndexes.inFront}
-      >
-        UP
-      </Scrollback>
+      <Scrollback bottom={20} left={20} />
```
Comment (Jbarget) [inline src/components/Projects.tsx:39]: very nice

**Review by Jbarget — CHANGES_REQUESTED** [review]
sort this out at the same time pleaseeeee :) 

image

**Comment by sssoja** [comment]
> sort this out at the same time pleaseeeee :)
> 
wooops!

**Review by sssoja — COMMENTED** [review]
Started a review by mistake. Ignore submit.

**Review by Jbarget — APPROVED** [review]

**Pull request merged**